scenario/AllowBySigScenario.ts

```diff
@@ -10,7 +10,7 @@ scenario(
     expect(await comet.isAllowed(albert.address, betty.address)).to.be.false;
 
     const nonce = await comet.userNonce(albert.address);
-    const expiry = (await world.timestamp()) + 1_000;
+    const expiry = (await world.timestamp()) + 10_000;
```

Comment:
This scenario (`Comet#allowBySig > allows a user to authorize a manager by signature`) fails inconsistently when run in CI.

This it the first scenario in the first scenario file (alphabetically).

Is it possible that the value of `world.timestamp` is returning the last mined block (created when World was first created by the Worker) and that by the time this scenario is executed, the next mined block is more than 1000 blocks past that initial block?

I wonder if it would solve this problem to mine a block before getting the world.timestamp value, instead of bumping the expiry to a very large number.

Reply:
Good call out. I think you might be right about mining a block being a potential fix for this.

Reply:
Great thought!

Reply:
Cherry-picked your change, Jared: https://github.com/compound-finance/comet/pull/569/commits/a0cb7876c48e2bbbad88f02dd5ef20a8c4f6dfa3